authentication/auth.go

```diff
@@ -219,11 +339,18 @@ func launcherSchemaFromURL(url string) (launcherSchema surveys.LauncherSchema, e
 		schemaName = schema.SchemaName
 	}
 
-	log.Println("Quicklaunch schema_name set to: ", schemaName)
+	surveyType := ""
+
+	if schema.SurveyType == "social" {
+		surveyType = "social"
+	} else {
+		surveyType = "business"
+	}
```

Comment:
This wouldn't work for anything else. At this point you already have access to the schema JSON itself, so this should be set based on `theme` inside the schema.

Reply:
I removed this now and am just using schema.SurveyType which comes from the schema.